UnitTests/CardPaymentsTests/ConfirmPaymentSourceRequest_Tests.swift

```diff
@@ -73,4 +74,30 @@ class ConfirmPaymentSourceRequest_Tests: XCTestCase {
         XCTAssertEqual(confirmPaymentSourceRequest.method, expectedMethod)
         XCTAssertEqual(confirmPaymentSourceRequest.headers, expectedHeaders)
     }
+
+    func testEncodingFailure_throws_EncodingError() throws {
+        let mockOrderID = "mockOrderID"
+        let card = Card(
+            number: "4032036247327321",
+            expirationMonth: "11",
+            expirationYear: "2024",
+            securityCode: "222"
+        )
+        let cardRequest = CardRequest(orderID: mockOrderID, card: card)
+        
+        let failingEncoder = FailingJSONEncoder()
+        
+        XCTAssertThrowsError(
+            try ConfirmPaymentSourceRequest(
+            accessToken: "fake token",
+            cardRequest: cardRequest,
+            encoder: failingEncoder)
```

Comment:
```suggestion
            try ConfirmPaymentSourceRequest(
                accessToken: "fake token",
                cardRequest: cardRequest,
                encoder: failingEncoder
            )
```

Reply:
addressed in 8276e5e070ada929f494281e93582e54d75475a8